pkg/build/api/validation/validation_test.go

```diff
@@ -319,6 +321,40 @@ func TestValidateBuildParameters(t *testing.T) {
 	}
 }
 
+func TestValidateBuildParametersSuccess(t *testing.T) {
```

Comment:
DONE. This test now exercises validation success in case the original Image field is non-empty.